packages/python/vc_init.py

```diff
@@ -82,14 +82,26 @@ def vc_handler(event, context):
         not inspect.iscoroutinefunction(__vc_module.app.__call__)
     ):
         print('using Web Server Gateway Interface (WSGI)')
+        from io import BytesIO
         from urllib.parse import urlparse
-        from werkzeug._compat import BytesIO
-        from werkzeug._compat import string_types
-        from werkzeug._compat import to_bytes
-        from werkzeug._compat import wsgi_encoding_dance
         from werkzeug.datastructures import Headers
         from werkzeug.wrappers import Response
 
+        string_types = (str,)
+        def to_bytes(x, charset=sys.getdefaultencoding(), errors="strict"):
+            if x is None:
+                return None
+            if isinstance(x, (bytes, bytearray, memoryview)):  # noqa
```

Comment:
What is `noqa`?

Reply:
noqa is like the python version of eslint-ignore

https://github.com/PyCQA/pycodestyle/issues/476#issuecomment-179553820